code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -237,11 +237,6 @@
 			M.emote("scream")
 	..()
 
-/datum/reagent/medicine/silver_sulfadiazine/on_mob_life(mob/living/M)
-	M.adjustFireLoss(-2*REM, 0)
-	..()
-	. = 1
-
```

Comment:
Isn't this just removing silver_sulfadiazine instead of nerfing it? If you do this, you'll need to rip out the other places where it's called on to prevent issues.

Reply:
No, that's just removing silver_sulfadine's on mob life.

Reply:
Ah, my mistake.

Reply:
No, you're right @UndeadNappist . On Mob Life is the effects of the drug while it's in the system.

Reply:
https://github.com/BadDeathclaw/TG-Claw/blob/master/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L227